test/job.js

```diff
@@ -169,15 +171,33 @@ describe('/CREATE, UPDATE, DELETE AND GET JOB', () => {
         });
     });
   });
+
+  describe('/GET Get all of jobs', () => {
+    it('return status 200 and a list all of jobs in json file when get the jobs successful!', (done) => {
+      chai
+        .request(app())
+        .get('/api/jobs')
+        .end((err, response) => {
+          response.should.have.status(200);
+          response.body.should.have
+            .property('message')
+            .eql('Get all jobs successful, list of jobs is below!');
+          expect(response.body.jobs).to.be.a.jsonObj();
```

Comment:
If we don't check each job information details, how can we know the function is correct? In the other hands, `expect(response.body.jobs).to.be.a.jsonObj()` this expectation doesn't make sure the result is correct, right?

Reply:
Fixed.